kafka.env

```diff
@@ -1,11 +1,19 @@
-ALLOW_PLAINTEXT_LISTENER=yes
-KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
-KAFKA_CFG_LISTENERS=INSIDE://:9093,OUTSIDE://:9092
-KAFKA_CFG_ADVERTISED_LISTENERS=INSIDE://:9093,OUTSIDE://kafka:9092
-KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT
+# KRaft settings
+KAFKA_CFG_NODE_ID=0
+KAFKA_CFG_PROCESS_ROLES=controller,broker
+KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093
+# Listeners
+KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093
+KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://:9092
+KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT
+KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
+KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT
+# Kafka settings
 KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE=false
-KAFKA_CFG_INTER_BROKER_LISTENER_NAME=INSIDE
 KAFKA_CFG_LOG_RETENTION_BYTES=1073741824
+# Kafka default value: 1 GB. TB value: ~27MB
```

Comment:
actually I see ~250MB
Probably here is an old mistake (not yours) same as on CE

Reply:
@smatvienko-tb what do you think is better in this case? Reset the value to ~27MB or update the comment back to ~250MB?

Reply:
@ShvaykaD please take a look on the number it Is one digit more than appears.
268435456/1024/1024=256 MB

Reply:
I have already changed to 25 MB on CE